app/Http/Controllers/Auth/LoginController.php

```diff
@@ -38,6 +40,22 @@ public function __construct() {
         $this->middleware('guest')->except('logout');
     }
 
+    public function login(Request $request): RedirectResponse {
+        $validated = $request->validate(['login'    => 'required',
+                                         'password' => 'required',
```

Comment:
Maybe we can add a `max:255` to login. E-Mails are limited to 255 and usernames to 25.
and `min:8` for password would be nice, too.